### ISSUE TITLE
More metrics

### ISSUE THREAD

**Issue by samriang** [deathowl/kafka_exporter#1]
Hi there.
Do you have any plans for developing this exporter? 
Increasing number of metrics, supporting latest kafka versions and so on.

**Comment by deathowl** [comment]
Yeah totally. If you have something more exact in mind don't hesitate to open the related issues, so I can work on those ideas
